frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx

```diff
@@ -200,12 +200,12 @@ class Visualization extends PureComponent {
     const question = new Question(card, metadata);
 
     // Datasets in QB should behave as raw tables opened in simple mode
-    // composeDataset replaces the dataset_query with a clean query using the dataset as a source table
+    // composeQuestion replaces the dataset_query with a clean query using the dataset as a source table
     // Ideally, this logic should happen somewhere else
```

Comment:
Does this still hold true, or we can remove the last line in the comment?

Reply:
Still true